ext/js/dictionary/dictionary-importer.js

```diff
@@ -615,8 +615,6 @@ export class DictionaryImporter {
      */
     _convertTermBankEntryV1(entry, dictionary) {
         let [expression, reading, definitionTags, rules, score, ...glossary] = entry;
-        expression = this._normalizeTermOrReading(expression);
-        reading = this._normalizeTermOrReading(reading.length > 0 ? reading : expression);
```

Comment:
Is removing this ternary intended? Should it instead be:
```suggestion
        reading = reading.length > 0 ? reading : expression;
```